src/test/features/ccj/routes/date-of-birth.ts

```diff
@@ -0,0 +1,75 @@
+import { expect } from 'chai'
+import * as request from 'supertest'
+import * as config from 'config'
+
+import { attachDefaultHooks } from '../../../routes/hooks'
+import '../../../routes/expectations'
+
+import { Paths } from 'ccj/paths'
+
+import { app } from '../../../../main/app'
+
+import * as idamServiceMock from '../../../http-mocks/idam'
+import * as claimStoreServiceMock from '../../../http-mocks/claim-store'
+import * as draftStoreServiceMock from '../../../http-mocks/draft-store'
+import { checkAuthorizationGuards } from './checks/authorization-check'
+
+const cookieName: string = config.get<string>('session.cookieName')
+
+const dateOfBirthPage = Paths.dateOfBirthPage.uri.replace(':externalId', 'b17af4d2-273f-4999-9895-bce382fa24c8')
+
+describe('CCJ - defendant date of birth', () => {
+  attachDefaultHooks()
+
+  describe('on GET', () => {
+    checkAuthorizationGuards(app, 'get', dateOfBirthPage)
+
+    context('when user authorised', () => {
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor(1, 'cmc-private-beta')
+      })
+
+      it('should render page when everything is fine', async () => {
```

Comment:
Missing tests for:
- failure in retrieval CCJ draft
- eligible CCD guard

The same comment applies for `POST` endpoint.